modules/apps/message-boards/message-boards-editor-configuration/src/main/java/com/liferay/message/boards/editor/configuration/internal/MBAttachmentHTMLEditorConfigContributor.java

```diff
@@ -42,7 +42,7 @@
  */
 @Component(
 	property = {
-		"editor.name=ckeditor", "editor.name=ckeditor_bbcode",
+		"editor.name=ckeditor_bbcode", "editor.name=ckeditor_classic",
```

Comment:
I'm wondering if we should also keep `ckeditor` in here, just in case... I know the value is mostly hardcoded, but it won't hurt in case someone somewhere manages to go back to `ckeditor` instead, so the ConfigContributor continues to apply. Same for the OptionsContributors